src/pages/ApplicationSettings.vue

```diff
@@ -2,45 +2,58 @@
   name: Application Settings
 </router>
 <template>
-  <path-management-selector
-    :value="pathManagementStrategy"
-    @input="writeSettings"
-  />
+  <div>
+    <path-management-selector
+      :value="pathManagementStrategy"
+      @input="onPathMananagementChange"
+    />
+    <checkbox
```

Comment:
+1 with what Jan said -- some kind of division is needed to put the "Allow administrative access" checkbox in a separate visual section.